mongoCollections.ts

```diff
@@ -0,0 +1,18 @@
+import { dbConnection } from "./mongoConnection";
+
+// This will allow you to have one reference to each collection per app
+function getCollectionFn(collection: any) {
```

Comment:
`collection: any` any should probably be changed to a string literal union - see `alignment` in `printTypes` here: https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#literal-types
If we change this to  string literal union, it will help make sure that we only try to get collections which really exist - you can also use the same string literal union as the Key type for `mongoCollections` below.

Reply:
Ahhh that's a great solution. I wasn't sure how to approach this, thank you!